lib/customer.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'order'
+
+class Customer
+  include Menu
+  include Text
+
+  def check_menu
+    display_menu
+  end
+
+  def verify_and_pay(order, total_price)
+    if order.order_total(total_price) == true
+      self.order_complete
+    else
+      fail 'Payment failed, please pay correct total.'
+    end
+  end
+
+  def order_complete
+    send_text(text_content)
```

Comment:
I'm not sure `Customer` should be sending texts. That seems like the job of something else.

You use composition to `include` the `Text` module, but I think you should have a `Text` class and inject it into `order_complete`.